osmosis-1/osmosis.zone_assets.json

```diff
@@ -5082,6 +5082,28 @@
       "override_properties": {
         "use_asset_name": true
       }
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1xu0gk9aggv79597xwazyfzaggv2pze9z7cq3p9p72tkkux9a7xaqufa792/BVT",
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-09-07T00:00:00Z",
+      "_comment": "Banana Vault Token - Peelworks Factory II",
+      "categories": [
+        "defi",
```

Comment:
how are these defi?

Reply:
It's a tokenized vault protocol.

Reply:
For context: https://forum.osmosis.zone/t/upload-banana-vaults-contracts/2484